services/ui-src/src/components/pages/ReviewSubmit/ReviewSubmitPage.tsx

```diff
@@ -197,6 +209,95 @@ const ReadyToSubmit = ({
   );
 };
 
+const AdminReview = ({
+  reviewVerbiage,
+  submitForm,
+  submitting,
+}: AdminReviewProps) => {
+  const { review } = reviewVerbiage;
+  const { adminInfo } = review;
+  const adminModal1 = useDisclosure();
+  const adminModal2 = useDisclosure();
+
+  return (
+    <Flex sx={sx.contentContainer} data-testid="ready-view">
+      <Box sx={sx.adminLeadTextBox}>
+        <Box sx={sx.infoTextBox}>
+          <Text sx={sx.infoHeading}>{adminInfo.header}</Text>
+          <Text>{parseCustomHtml(adminInfo.info)}</Text>
+        </Box>
+      </Box>
+      <Flex sx={sx.adminSubmitContainer}>
```

Comment:
These buttons should be disabled if the Work Plan is unlocked - and if the WP hasn't been submitted yet, the work plan is unlocked